### PR TITLE
Implement Darkmode on Governance and Outreachy page

### PR DESCRIPTION
**Pull request by oyenuga17**
@SaySayo please help review

### CODE REVIEW

File: src/ocamlorg_frontend/pages/governance.eml [hunk 1]
```diff
@@ -3,19 +3,14 @@ module MemberSet = Set.Make (Data.Governance.Member)
 let count_members (team : Data.Governance.team) =
    MemberSet.cardinal (MemberSet.of_list (team.members @ List.concat (List.map (fun (team : Data.Governance.team) -> team.members) team.subteams)))
 
-let render_team_card (team : Data.Governance.team) member_label btn_class =
+let render_work_group_card (team : Data.Governance.team) member_label btn_class =
   <div class="flex-col">
-    <div class="text-2xl mb-5 font-semibold flex items-center">
+    <div class="text-2xl mb-2 font-semibold flex items-cente text-white dark:text-dark-title">
```
Comment (SaySayo) [inline src/ocamlorg_frontend/pages/governance.eml]:
```suggestion
    <div class="text-2xl mb-2 font-semibold flex items-center text-white dark:text-dark-title">
```

File: src/ocamlorg_frontend/pages/governance.eml [hunk 2]
```diff
@@ -24,71 +19,60 @@ let render_team_card (team : Data.Governance.team) member_label btn_class =
...
-        The <a href="<%s Url.governance_policy %>">OCaml Governance Policy</a> does not apply to them. Participation is welcome and encouraged.</div>
-      <div class="grid grid-cols-1 md:grid-cols-2 gap-20 mt-16">
-        <% working_groups |> List.iter (fun (wg : Data.Governance.team) -> %>
-        <%s! render_team_card wg "organiser(s)" "btn-ghost" %>
-        <% ); %>
+<div class="w-full governance-blue-gradient dark:dark-governance-blue-gradient">
+  <div class="container-fluid flex justify-between items-center py-7 px-8">
+      <div class="flex flex-col w-full">
+        <div class="md:max-w-[80%]">
+          <h2 class="font-bold text-2xl text-white dark:text-dark-title mb-2">Working groups</h2>
```
Comment (SaySayo) [inline src/ocamlorg_frontend/pages/governance.eml:65]:
```suggestion
          <h2 class="font-bold text-2xl text-title dark:text-dark-title mb-2">Working groups</h2>
```

Comment (SaySayo) [reply src/ocamlorg_frontend/pages/governance.eml:65]: Isn't the dark mode implementation of `text-white` `text-dark-white` and for `text-title`, `text-dark-title`? I notice you used them interchangeably multiple times across the code

**Review by SaySayo — CHANGES_REQUESTED** [review]

File: src/ocamlorg_frontend/pages/governance_team.eml [hunk 3]
```diff
@@ -5,60 +5,59 @@ let contact_icon (kind : Data.Governance.contact_kind) = match kind with
...
-  <div class="flex gap-x-2">
-    <img src="https://github.com/<%s member.github %>.png" alt="" class="h-32 w-32 mr-3 inline-block rounded-xl">
+  <a href="https://github.com/<%s member.github %>" class="flex gap-x-2 p-4 card dark:dark-card">
+    <img src="https://github.com/<%s member.github %>.png" alt="" class="h-32 w-32 mr-3 inline-block rounded-full">
     <div>
-      <div class="text-white text-lg font-semibold mt-3"><%s member.name %></div>
-      <div class="text-sm text-white text-opacity-60"><%s member.role %></div>
-      <div class="mt-8 text-white text-opacity-60 items-center flex gap-1">
+      <div class="text-title dark:text-dark-title text-xl mt-3"><%s member.name %></div>
+      <div class="text-sm text-content dark:text-dark-content text-opacity-60"><%s member.role %></div>
```
Comment (sabine) [inline src/ocamlorg_frontend/pages/governance_team.eml]:
```suggestion
      <div class="text-sm text-content dark:text-dark-content"><%s member.role %></div>
```

File: src/ocamlorg_frontend/pages/governance_team.eml [hunk 4]
```diff
@@ -5,60 +5,59 @@ let contact_icon (kind : Data.Governance.contact_kind) = match kind with
...
     <% (match team.dev_meeting with | None -> () | Some dev_meeting -> %>
-    <div class="py-4 px-6 mt-6 bg-white inline-block rounded-xl">
-      <h4 class="font-semibold mb-4 text-2xl">Dev Meeting</h4>
+    <div class="py-4 px-6 mt-6 bg-white hover:shadow-lg rounded-2xl border inline-block dark:bg-dark-card">
+      <h4 class="font-semibold mb-4 text-2xl text-title dark:text-dark-title">Dev Meeting</h4>
       <%s! render_dev_meeting dev_meeting "" %>
     </div>
     <% ); %>
-    <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-12 mt-6">
+    <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-12 mt-6 mb-4">
```
Comment (sabine) [inline src/ocamlorg_frontend/pages/governance_team.eml]:
```suggestion
    <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-10 mt-6 mb-4">
```

**Review by sabine — COMMENTED** [review]
Here's some differences between the Figma document and this PR:

|Figma|This PR|Note|
|-|-|-|
|Screenshot from 2024-02-16 10-19-18|Screenshot 2024-02-16 at 10-19-34 Governance|padding/spacing|
|image|image|space above below separator is uneven|
|image|Screenshot from 2024-02-16 10-47-09|card in this PR, no card in Figma, spacing issue between the links and the text|

* Card should not have a hover effect that implies that it's clickable when it is not clickable - it seems only the details link is clickable. However, this link has a very small click area and visual spacing between such a small link and other elements should be created by adding padding to the link - this increases the click area, which in turn enables people with motor difficulties to actually click this.

* button here has no visual indicator on hover, this should probably be using one of the button component styles instead of being tailwind classes on the `a` element:
image

**Comment by oyenuga17** [comment]
@sabine I am now ready for a review thank you.

**Comment by sabine** [comment]
Merging, thanks @oyenuga17 !

**Pull request merged**